app/code/Magento/Customer/Controller/Adminhtml/Group/Save.php

```diff
@@ -46,67 +58,55 @@ public function __construct(
             $resultForwardFactory,
             $resultPageFactory
         );
+        $this->dataPersistor = $dataPersistor ?:
+            $this->_objectManager->get(DataPersistorInterface::class);
     }
 
     /**
-     * Store Customer Group Data to session
-     *
-     * @param array $customerGroupData
-     * @return void
-     */
-    protected function storeCustomerGroupDataToSession($customerGroupData)
```

Comment:
please keep this function and mark as deprecated

Reply:
done